types/decimal.go

```diff
@@ -323,6 +323,32 @@ func (d Dec) RoundInt() Int {
 
 //___________________________________________________________________________________
 
+// similar to chopPrecisionAndRound, but always rounds down
+func chopPrecisionAndTruncate(d *big.Int) *big.Int {
+	return d.Quo(d, precisionReuse)
+}
+
+func chopPrecisionAndTruncateNonMutative(d *big.Int) *big.Int {
```

Comment:
Do we really need `chopPrecisionAndTruncate`? Can't we just have `chopPrecisionAndTruncateNonMutative` and rename it to `chopPrecisionAndTruncate`

Reply:
Not a big deal either way, but only `chopPrecisionAndTruncateNonMutative` uses `chopPrecisionAndTruncate`.

Reply:
I just wanted to stay consistent with how rounding is done, I'm going to leave